### ISSUE TITLE
Running program error

### ISSUE THREAD

**Issue by QiuJianL** [sugiyamath/bert_openie#1]
Hello, I have the following problems during the running of the program. Have you encountered any problems?

INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into ./output\model.ckpt.
2019-05-21 17:21:01.355459: E tensorflow/core/kernels/check_numerics_op.cc:185] abnormal_detected_host @0000000203A02800 = {1, 0} Found Inf or NaN global norm.
INFO:tensorflow:Error recorded from training_loop: Found Inf or NaN global norm. : Tensor had NaN values
         [[node VerifyFinite/CheckNumerics (defined at D:\Workproject\5.10\train\bert_openie-master\bert\optimization.py:74)  = CheckNumerics[T=DT_FLOAT, message="Found Inf or NaN global norm.", _device="/job:localhost/replica:0/task:0/device:GPU:0"](global_norm/global_norm)]]

Caused by op 'VerifyFinite/CheckNumerics', defined at:
  File "run_openie.py", line 1384, in <module>
    tf.app.run()
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\platform\app.py", line 125, in run
    _sys.exit(main(argv))
  File "run_openie.py", line 1307, in main
    estimator.train(input_fn=train_input_fn, max_steps=num_train_steps)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\contrib\tpu\python\tpu\tpu_estimator.py", line 2403, in train
    saving_listeners=saving_listeners
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\estimator\estimator.py", line 354, in train
    loss = self._train_model(input_fn, hooks, saving_listeners)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\estimator\estimator.py", line 1207, in _train_model
    return self._train_model_default(input_fn, hooks, saving_listeners)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\estimator\estimator.py", line 1237, in _train_model_default
    features, labels, model_fn_lib.ModeKeys.TRAIN, self.config)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\contrib\tpu\python\tpu\tpu_estimator.py", line 2195, in _call_model_fn
    features, labels, mode, config)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\estimator\estimator.py", line 1195, in _call_model_fn
    model_fn_results = self._model_fn(features=features, **kwargs)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\contrib\tpu\python\tpu\tpu_estimator.py", line 2479, in _model_fn
    features, labels, is_export_mode=is_export_mode)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\contrib\tpu\python\tpu\tpu_estimator.py", line 1259, in call_without_tpu
    return self._call_model_fn(features, labels, is_export_mode=is_export_mode)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\contrib\tpu\python\tpu\tpu_estimator.py", line 1533, in _call_model_fn
    estimator_spec = self._model_fn(features=features, **kwargs)
  File "run_openie.py", line 675, in model_fn
    total_loss, learning_rate, num_train_steps, num_warmup_steps, use_tpu)
  File "D:\Workproject\5.10\train\bert_openie-master\bert\optimization.py", line 74, in create_optimizer
    (grads, _) = tf.clip_by_global_norm(grads, clip_norm=1.0)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\ops\clip_ops.py", line 265, in clip_by_global_norm
    "Found Inf or NaN global norm.")
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\ops\numerics.py", line 47, in verify_tensor_all_finite
    verify_input = array_ops.check_numerics(t, message=msg)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_array_ops.py", line 817, in check_numerics
    "CheckNumerics", tensor=tensor, message=message, name=name)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\util\deprecation.py", line 488, in new_func
    return func(*args, **kwargs)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3274, in create_op
    op_def=op_def)
  File "C:\Users\Administrator\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 1770, in __init__
    self._traceback = tf_stack.extract_stack()
